includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -60,4 +78,39 @@ public static function get_plugin_basename_from_input( $plugin_slug ) {
 			)
 		);
 	}
+
+	/**
+	 * Initializes the runner classes.
+	 *
+	 * @since n.e.x.t
+	 */
+	public static function initialize_runner() {
+		$runners = array(
+			new CLI_Runner(),
+			new AJAX_Runner(),
+		);
+
+		foreach ( $runners as $runner ) {
+			if ( $runner->is_plugin_check() ) {
+				static::$cleanup = $runner->prepare();
```

Comment:
This cleanup isn't used anywhere and also isn't accessible from the outside. I'm not sure how to best handle this yet, we should probably revisit once we are closer to having the full architecture.

Some ideas:
* Maybe we add a method to this class to run the cleanup on demand from somewhere else.
* Or maybe we immediately hook it on the `shutdown` action here so that it will run automatically.

Maybe best to add a `TODO` comment somewhere here or open a follow up issue so that we don't forget about completing this part.

Reply:
Thanks @felixarntz I've created a new issue to address this here https://github.com/10up/plugin-check/issues/85